internal/cli/job_inspect.go

```diff
@@ -112,6 +112,8 @@ func (c *JobInspectCommand) Run(args []string) int {
 		op = "Init"
 	case *pb.Job_PipelineStep:
 		op = "PipelineStep"
+	case *pb.Job_DestroyProject:
+		op = "DestroyProject"
```

Comment:
I believe similar changes need to be made in [`cli/task_list.go`](https://github.com/hashicorp/waypoint/blob/420f95a7f08178ed88b5ce549fb04b45b443e1fd/internal/cli/task_list.go#L102) and [`cli/task_inspect.go`](https://github.com/hashicorp/waypoint/blob/420f95a7f08178ed88b5ce549fb04b45b443e1fd/internal/cli/task_inspect.go#L169), unfortunately :/